vite.config.js

```diff
@@ -8,7 +8,7 @@ export default defineConfig({
   plugins: [reactRefresh(), cssInjectedByJsPlugin()],
   build: {
     lib: {
-      entry: path.resolve(__dirname, 'src', 'materialUIMachineOptions.js'),
+      entry: path.resolve(__dirname, 'src', 'bootstrapUIMachineOptions.js'),
       name: 'materialUIMachineOptions',
       formats: ['es'],
       fileName: 'materialUIMachineOptions.js'
```

Comment:
Maybe change name to `name: 'bootstrapUIMachineOptions'  and we can delete `fileName` .  Default should be good.  Test with `npm run build` and check dist directory.